Helpers/ShareOfCheckout/SOCHelper.php

```diff
@@ -13,11 +13,14 @@
 use Magento\Framework\App\Helper\Context;
 use Magento\Store\Model\ScopeInterface;
 
-class ShareOfCheckoutHelper extends AbstractHelper
+class SOCHelper extends AbstractHelper
 {
     const SHARED_ORDER_STATES = ['processing', 'complete'];
     const SHARE_CHECKOUT_ENABLE_KEY = 'share_checkout_enable';
     const SHARE_CHECKOUT_DATE_KEY = 'share_checkout_date';
```

Comment:
```suggestion
    const SOC_ENABLE_KEY = 'soc_enabled';
    const SOC_DATE_KEY = 'soc_date';
```

be careful with `enable` that becomes `enabled` in your `system.xml` 😉